google/resource_compute_global_forwarding_rule_test.go

```diff
@@ -65,6 +67,36 @@ func TestAccComputeGlobalForwardingRule_update(t *testing.T) {
 	})
 }
 
+func TestAccComputeGlobalForwardingRule_ipv6(t *testing.T) {
+	var frule computeBeta.ForwardingRule
+
+	fr := fmt.Sprintf("forwardrule-test-%s", acctest.RandString(10))
+	proxy1 := fmt.Sprintf("forwardrule-test-%s", acctest.RandString(10))
+	proxy2 := fmt.Sprintf("forwardrule-test-%s", acctest.RandString(10))
+	backend := fmt.Sprintf("forwardrule-test-%s", acctest.RandString(10))
+	hc := fmt.Sprintf("forwardrule-test-%s", acctest.RandString(10))
+	urlmap := fmt.Sprintf("forwardrule-test-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeGlobalForwardingRuleDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccComputeGlobalForwardingRule_ipv6(fr, proxy1, proxy2, backend, hc, urlmap),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeBetaGlobalForwardingRuleExists(
```

Comment:
Checked IPV4 on the `_basic` case, `IpVersion` is `""` for this resource by default, but that means `IPV4`.